FrontEnd/src/HomePage/PostFullDisplay.jsx

```diff
@@ -77,28 +80,46 @@ function PostFullDisplay({
             <div className="fullDisplayDetails">
               <h3 className="fullDisplayTitle">{post.serviceTitle}</h3>
               <p className="fullDisplayPrice">
-                Price Range: {post.servicePrice}
+                <strong>Price Range:</strong> {post.servicePrice}
               </p>
               <p className="fullDisplayDescription">
-                Description: {post.serviceDescription}
-              </p>
-              <button onClick={handleBookingFormOpen}>Book Now</button>
-              <span>
-                {favorited ? (
-                  <i
-                    className="fa-solid fa-heart hearted"
-                    onClick={() => handleNotification(1)}
-                  />
-                ) : (
-                  <i
-                    className="fa-regular fa-heart"
-                    onClick={() => handleNotification(1)}
-                  />
-                )}
-              </span>
-              <p className="fullDisplayTimestamp">
-                Created at: {createdAt.toLocaleString()}
+                <strong>Description:</strong> {post.serviceDescription}
               </p>
+              {post.vendorUID !== userUID && (
+                <div>
+                  <button
+                    onClick={handleBookingFormOpen}
+                    className="bookButton"
+                  >
+                    Book Now
+                  </button>
+                  <span>
+                    {favorited ? (
```

Comment:
You could conditionally apply one class instead of repeating the entire code block:

```
<i
    className={`fa-solid fa-heart{favorited? " hearted": ""}`}
    onClick={() => handleNotification(1)}
  />
```

Reply:
Can you expatiate on this? I don't quite get it

Reply:
You can move the condition check to the class name.

Instead of this

```
favorited ? <i className="fa-solid fa-heart"  ... /> :  <i className="fa-solid fa-heart hearted"  ... /> 
```
it will be
```
<i className={`fa-solid fa-heart{favorited? " hearted": ""}`} ...  />
```

Less code repetition this way.